pilot/pkg/model/jwks_resolver.go

```diff
@@ -93,6 +92,9 @@ var (
 
 	// JwtPubKeyRefreshInterval is the running interval of JWT pubKey refresh job.
 	JwtPubKeyRefreshInterval = features.PilotJwtPubKeyRefreshInterval
+
+	// channel for making jwksuri request aynsc
+	jwksuriChannel = make(chan jwtKey, 5)
```

Comment:
Slightly worried about size of 5.. users probably don't have many more but if they do it can block everything